### PR TITLE
Update portal-ui maintenance for Dec 2023 outage.

### PR DESCRIPTION
**Pull request by lchoy**
Updates portal UI messaging for Dec 19-20, 2023 planned outage.

### CODE REVIEW

File: nginx/html/portal-ui-maintenance/index.html [hunk 1]
```diff
@@ -1 +1 @@
-<!doctype html><html><head><meta charset="utf-8"><meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no"><title>Maintenance | HuBMAP</title><link rel="icon" href="/favicon.ico"></head><body><div id="react-content"></div><script src="/maintenance.bundle.js"></script></body></html>
\ No newline at end of file
+<!doctype html><html><head><meta charset="utf-8"/><meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no"/><title>Maintenance | HuBMAP</title><script>const sentryEnv = '{{ sentry_env }}';</script><link rel="icon" href="favicon.ico"><script defer="defer" src="maintenance.bundle.js"></script></head><body><div id="react-content"></div></body></html>
```
Comment (yuanzhou) [inline nginx/html/portal-ui-maintenance/index.html:1]: What is this `sentry_env`? Do I need to do anything specially?

**Review by yuanzhou — COMMENTED** [review]
@lchoy looks like the main update is to show 
```
Portal is unavailable for scheduled maintenance from December 19th to December 20th, 2023.
```
Is this configurable or just hardcoded for one-time use? Maybe I can just update the minified file with this message without doing anything to this PR?

File: nginx/html/portal-ui-maintenance/index.html (hunk 1 above)
Comment (lchoy) [reply nginx/html/portal-ui-maintenance/index.html:1]: `sentry_env` is a variable for Portal tracking. There's nothing that needs to be done for that. 


This PR is an updated hardcoded message for one-time use. We'll address making the messaging configurable in a later update.

**Review by yuanzhou — APPROVED** [review]
@lchoy I'm gonna merge this PR and use the updated maintenance info for the 19-20 PSC outage. 

After this one-time use, we'll need to either change the message (only the message wording) back to a generic sentence or make it configurable for later use.

**Pull request merged**